### PR TITLE
Doc: add instructions to use the Fugaku supercomputer (Riken)

### PR DESCRIPTION
**Pull request by lucafedeli88**
This PR adds some documentation to compile and run WarpX on the Fugaku supercomputer

### CODE REVIEW

File: Docs/source/install/hpc/fugaku.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
+.. _building-fugaku:
+
+Fugaku (Riken)
+==================
```
Comment (ax3l) [inline Docs/source/install/hpc/fugaku.rst]: RST is a bit picky here:

```suggestion
Fugaku (Riken)
==============
```

File: Docs/source/install/hpc/fugaku.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+.. _running-cpp-fugaku:
+
+Running
+-------
+
+.. _running-cpp-fugaku-A64FX-CPUs:
+
+A64FX CPUs
+^^^^^^^^^^^^^^^^^^^^^
```
Comment (ax3l) [inline Docs/source/install/hpc/fugaku.rst]:
```suggestion
A64FX CPUs
^^^^^^^^^^
```

**Review by ax3l — COMMENTED** [review]
Awesome, thanks! Forgot we never moved details over after GB.

Don't forget to add Fugaku in `Docs/source/install/hpc.rst` so it gets linked and rendered :)

**Comment by lucafedeli88** [comment]
> Awesome, thanks! Forgot we never moved details over after GB.
> 
> Don't forget to add Fugaku in `Docs/source/install/hpc.rst` so it gets linked and rendered :)

right! I always forget to do that!

**Review by ax3l — APPROVED** [review]
Nice, thanks a lot! :)

**Pull request merged**